docs/redoc/master/openapi.json

```diff
@@ -7358,12 +7358,14 @@
             "description": "Point version",
             "type": "integer",
             "format": "uint64",
-            "minimum": 0
+            "minimum": 0,
+            "example": 1
           },
           "score": {
             "description": "Points vector distance to the query vector",
             "type": "number",
-            "format": "float"
+            "format": "float",
+            "example": 1.100000023841858
```

Comment:
`1.1` have no nice representation in f32, that's why the example is that ugly.

Please choose the different value